### ISSUE TITLE
timing of Practice teaching exercise

### ISSUE THREAD

**Issue by brownsarahm** [carpentries/instructor-training#1420]
the timing of the [first live coding practice](https://carpentries.github.io/instructor-training/17-live/index.html#practice-teaching) is inconsistent. the described actions total to 12 minutes : 3x(3 min teaching+ 1 min feedback) but the total time listed is 25.

**Comment by fiveop** [comment]
I agree that the math does not check out, but…

…with a little buffer between feedback and teaching as well as in the beginning, we are at around 15 minutes. In addition, this is an exercise participants are nervous about and have some questions. If the q&a before the actual exercise counts towards its duration the 25 minutes are justified.

**Comment by brownsarahm** [comment]
I agree that it usually *takes* 25 minutes in the schedule, but we could align the text better. It would help trainers set a timer on the breakout rooms more easily. 

Also I think noting that there is typically whole group discussion/debrief after.
